src/ConfigFactory.Core/ConfigModule.cs

```diff
@@ -132,4 +132,15 @@ public virtual bool Validate(out string? message, out ConfigProperty target)
         message = "Validation Successful";
         return true;
     }
+
+    /// <summary>
+    /// Default implementation of the <see cref="IConfigModule.Translate(string)"/>.
+    /// Returns the same same string.
```

Comment:
Typo in code documentation

```suggestion
    /// Default implementation of the <see cref="IConfigModule.Translate(string)"/>.<br/>
    /// Returns the <paramref name="input"/> string.
```

Reply:
Fixed